x-pack/plugins/maps/public/classes/sources/es_search_source/es_search_source.tsx

```diff
@@ -499,7 +499,7 @@ export class ESSearchSource extends AbstractESSource implements ITiledSingleLaye
     };
 
     searchSource.setField('query', query);
-    searchSource.setField('fields', this._getTooltipPropertyNames());
+    searchSource.setField('fieldsFromSource', this._getTooltipPropertyNames());
```

Comment:
These should all eventually move back to `setField('fields', [...])` which will now use search fields API under the hood.